### DISCUSSION TITLE
issues on windows 11

### DISCUSSION THREAD

**Discussion by HasturDev** [reflex-dev/reflex#1959] [Q&A]
in attempting to run this application just as the tutorial says. I hit an error on windows 11
Does this error mean I should pre-install node on my pc before running this application?
(.venv) C:\Users\(username)\Documents\PyATL\app_name>python -m reflex init --loglevel debug
Info: Overriding config value username with env var USERNAME=(username)
Info: Overriding config value username with env var USERNAME=(username)
─────────────────────────────── System Info ───────────────────────────────
Debug: Config file:
'C:\\Users\\(username)\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Pytho
n.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\
\reflex\\config.py'
Debug: Config: app_name='' loglevel=<LogLevel.INFO: 'info'>
frontend_port=3000 frontend_path='' backend_port=8000
api_url='http://localhost:8000' deploy_url='http://localhost:3000'
backend_host='0.0.0.0' db_url='sqlite:///reflex.db' redis_url=None
telemetry_enabled=True
bun_path='C:\\Users\\(username)\\AppData\\Local\\reflex\\bun\\bin\\bun'
cors_allowed_origins=['*'] tailwind={} timeout=120 next_compression=True
event_namespace=None frontend_packages=[] rxdeploy_url=None
username='(username)'
Warning: The path to the Node binary could not be found. Please ensure that
Node is properly installed and added to your system's PATH environment
variable.
Debug: Running command: [None, '-v']
Debug: [Reflex 0.2.9 with Python 3.11.6 (PATH:
C:\Users\(username)\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation
.Python.3.11_qbz5n2kfra8p0\python.exe)]
Debug: [Node None (Expected: 18.17.0) (PATH:None)]
Debug: [FNM 1.35.1 (Expected: 1.35.1) (PATH:
C:\Users\(username)\AppData\Local\reflex\fnm\fnm.exe)]
Debug: [OS Windows 10.0.22621]
Debug: Using package installer at: None
Debug: Using package executer at: None
────────────────────────── Initializing app_name ──────────────────────────
Debug: Skipping bun installation on Windows.
Warning: The path to the Node binary could not be found. Please ensure that
Node is properly installed and added to your system's PATH environment
variable.
Debug: Running command: ['powershell', '-Command', '&
"C:\\Users\\(username)\\AppData\\Local\\reflex\\fnm\\fnm.exe" install 18.17.0
--fnm-dir "C:\\Users\\i_miz\\AppData\\Local\\reflex\\fnm"']
Debug: Installing node
Debug: & : The term 'C:\Users\(username)\AppData\Local\reflex\fnm\fnm.exe' is
not
Debug: recognized as the name of a cmdlet, function, script file, or
operable
Debug: program. Check the spelling of the name, or if a path was included,
verify
Debug: that the path is correct and try again.
Debug: At line:1 char:3
Debug: + & "C:\Users\(username)\AppData\Local\reflex\fnm\fnm.exe" install
18.17.0 - ...
Debug: +   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Debug:     + CategoryInfo          : ObjectNotFound:
(C:\Users\(username)\...lex\fnm\f
Debug:    nm.exe:String) [], CommandNotFoundException
Debug:     + FullyQualifiedErrorId : CommandNotFoundException
Debug:
Installing node failed with exit code 1
& : The term 'C:\Users\(username)\AppData\Local\reflex\fnm\fnm.exe' is not
recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify
that the path is correct and try again.
At line:1 char:3
+ & "C:\Users\(username)\AppData\Local\reflex\fnm\fnm.exe" install 18.17.0 - ...
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\(username)\...lex\fnm\f
   nm.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Run with --loglevel debug  for the full log.

**Comment by masenf** [comment]
Reflex is supposed to use FNM to install a compatible version of node for you. For whatever reason, FNM is not able to be installed here. You could try deleting `C:\Users(username)\AppData\Local\reflex` and running `reflex init` and `reflex run` again.

I'm not able to reproduce on my win11 VM.

**Reply by HasturDev** [reply]
Looking at the issue further. There is no Reflex created in the local when running reflex init

**Reply by HasturDev** [reply]
Yeah I just looked through the local after doing a reflex init and found nothing
[list.txt](https://github.com/reflex-dev/reflex/files/13193213/list.txt)
